specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/MitreCoverage.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Security Insights",
+    "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+    "version": "2022-01-01-preview",
+    "x-ms-code-generation-settings": {
+      "name": "SecurityInsightsClient"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/mitreCoverage": {
+      "get": {
+        "tags": [
+          "MitreCoverage"
+        ],
+        "description": "Fetches Products Mitre Coverage from Data sources",
+        "operationId": "Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Products Mitre Coverages fetched",
+            "schema": {
+              "$ref": "#/definitions/ProductsMitreCoverageList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed."
+          }
+        },
+        "x-ms-examples": {
+          "Get": {
+            "$ref": "./examples/MitreCoverage/Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ProductMitreCoverageModel": {
+      "description": "A wrapper to the resource of type T, which contains Arm's first level fields such as id, name and type, with the resource properties",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/1.0/types.json#/definitions/ResourceWithEtag"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Gets or sets nested level of properties which contains the resource content",
+          "type": "object",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ProductMitreCoverageProperties"
+            }
+          ],
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ProductMitreCoverageProperties": {
+      "type": "object",
+      "properties": {
+        "productName": {
+          "description": "Gets Or sets the product name that covers a specific tactic and technique.",
```

Comment:
>Gets Or sets the  [](http://example.com/codeflow?start=26&length=17)

This should describe the property, not the operations. Your API doesn't support PUT anyway.

Reply:
Recommend more like "Product name that covers a specific tactic and technique."